lib/test-stream.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const Readable = require('stream').Readable;
+const testObservable = require('./test-observable');
+
+module.exports = class TestStream extends Readable {
+  constructor(test262Dir, options) {
+    options = Object.assign({}, options);
+    options.objectMode = true;
+    super(options);
+    this._test262Dir = test262Dir;
+    this._options = options;
+    this._observable = null;
```

Comment:
Update to Symbols per offline discussion—thanks!

Reply:
Sure thing